live-examples/css-examples/basic-box-model/overscroll-behavior-x.html

```diff
@@ -0,0 +1,54 @@
+
+    <div class="example-choice">
+        <pre><code class="language-css">overscroll-behavior-x: auto;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">overscroll-behavior-x: contain;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+    <div class="example-choice">
+        <pre><code class="language-css">overscroll-behavior-x: none;</code></pre>
+        <button type="button" class="copy hidden" aria-hidden="true">
+            <span class="visually-hidden">Copy to Clipboard</span>
+        </button>
+    </div>
+
+</section>
+
+<div id="output" class="output large hidden">
+    <section id="default-example" class="default-example">
+        <div class="example-container">
+            <div class="box">
+                This is a scrollable container. Michaelmas term lately over, and the Lord Chancellor sitting in Lincoln's Inn Hall. Implacable November weather. As much mud in the streets as if the waters had but newly retired from the face of the earth.
+                <br><br>
+                Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essentially unchanged.
+            </div>
+            <div id="example-element">
+                <div id="element-left">
+                    This is the inner container. Focus on this container, scroll to the right and when you reach the right edge keep scrolling.
+                    <p>If you have <code class="language-css">overscroll-behavior-x: 
+                        auto;</code> selected the outer container 
+                        will start to scroll.</p>
+                    If you have <code class="language-css">overscroll-behavior-x: 
+                        contain;</code> selected, the outer container will not scroll unless you move your cursor out of the inner container and try to perform scroll on the outer container. 
+
+                        **Note that this applies only to the x-axis. You should still be able to scroll past in the y-axis.
+                </div>
+                <div id="element-right">
+                    This is sample text to fill up space. Lorem ipsum dolor sit amet consectetur adipisicing elit. Eius veniam, animi laudantium quibusdam vitae cumque officiis in eos laborum molestiae quasi ex eaque minima architecto, inventore natus fuga tempora accusamus velit deserunt numquam, illo nemo labore accusantium. Earum nihil placeat sequi veritatis tempora, corrupti et necessitatibus quos! Rem, sed quod!
```

Comment:
This paragraph is way too long, it takes forever to scroll through it on the phone. Also visible content of the box should indicate that there is some additional content on the right. On many devices, scrollbar is not shown until scrolling through the content, so users can easily think that there is more text only at the bottom:
![image](https://user-images.githubusercontent.com/100634371/220114034-b78a82b2-ea48-4510-819e-1c61ee072e81.png)